modules/matlab/include/opencv2/matlab/bridge.hpp

```diff
@@ -522,6 +522,11 @@ class Bridge {
   Bridge& operator=(const Ptr_TonemapReinhard& ) { return *this; }
   Ptr_TonemapReinhard toPtrTonemapReinhard() { return Ptr_TonemapReinhard(); }
   operator Ptr_TonemapReinhard() { return toPtrTonemapReinhard(); }
+
+  // ---------------------------   Ptr_float   ----------------------
+  Bridge& operator=(const Ptr_float& ) { return *this; }
+  Ptr_float toPtrFloat() { return Ptr_float(); }
```

Comment:
BTW, Looks like this method always create new empty "Ptr", so statement below will not use "input value" at all:
```
Ptr_float lowerBound = inputs[4].empty() ? (Ptr_float) Ptr<float>() : inputs[4].toPtrFloat();
```